iphone/Maps/UI/PlacePage/Components/PlacePageHeader/CircleImageButton.swift

```diff
@@ -0,0 +1,52 @@
+final class CircleImageButton: UIButton {
+
+  private let circleImageView = UIImageView()
+
+  override init(frame: CGRect) {
+    super.init(frame: frame)
+    setupView()
+  }
+
+  @available(*, unavailable)
```

Comment:
What does it mean?

Reply:
It is used to remove this designated initializer (that is required to implement) from the public API so its impossible to initialize the instance using this initializer. 
But the system itself can use it while instantiating the objects from nibs.